recipe/yum_requirements.txt.bak

```diff
@@ -1,8 +0,0 @@
-# TODO get rid of this and use xorg package.
-#mesa-libGL-devel
```

Comment:
Note: Looks like this was left over from a merge conflict tool. Maybe it should be dropped?